underactuated.html

```diff
@@ -828,6 +828,23 @@ <h1>Honda's ASIMO vs. passive dynamic walkers</h1>
 
     </exercise>
 
+    <exercise><h1>(Trajectory Tracking for Fully-Actuated Systems)</h1>
```

Comment:
You shouldn't add the parens.  If we want them, then I can add them in the css file.

Reply:
OK, should we do that each exercise must have a title then? I think it makes sense overall, to keep thing ordered.